images-list

```diff
@@ -3,6 +3,10 @@
 # https://github.com/rancher/image-mirror/pull/27
 #
 # LIST FORMAT: <SOURCE> <DESTINATION> <TAG>
+amazon/aws-cli rancher/amazon-aws-cli 2.0.52
```

Comment:
Isn't this a duplicate of the next line?

Reply:
I tried to be in line with the existing definitions where each incoming `foo tag` apparently resulted in two lines:
> foo rancher/foo tag
foo rancher/mirrored-foo tag

Example:
> bats/bats rancher/bats-bats v1.1.0
bats/bats rancher/mirrored-bats-bats v1.1.0

Looking again it does not look to be perfectly matching, especially when multiple versions of an image are involved.
For example:
> banzaicloud/fluentd rancher/banzaicloud-fluentd v1.11.2-alpine-2
banzaicloud/fluentd rancher/banzaicloud-fluentd v1.11.4-alpine-1
banzaicloud/fluentd rancher/banzaicloud-fluentd v1.11.5-alpine-1
banzaicloud/fluentd rancher/banzaicloud-fluentd v1.11.5-alpine-9
banzaicloud/fluentd rancher/mirrored-banzaicloud-fluentd v1.11.5-alpine-9
banzaicloud/fluentd rancher/mirrored-banzaicloud-fluentd v1.11.5-alpine-12
banzaicloud/fluentd rancher/mirrored-banzaicloud-fluentd v1.11.5-alpine-21

There is no true pattern I can see. Sometimes it seems just older versions of the image have lines without the `mirrored-` prefix. In some cases there is overlap, versions with and without prefix. I now also see some only with prefix.

Ok. I will go with the request and remove the "duplicates" which do not have the prefix, leaving the lines containing `mirrored-`.

Is that explained somewhere I missed ?

Reply:
Duplicate removed with commit 0b45d071b646cc68472c0a2bb5aaf9fd118ddc09